std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -170,22 +170,18 @@ public static Column fromItemsNoDateConversion(String name, List<Object> items,
   public static Column fromRepeatedItem(String name, Value item, int repeat, ProblemAggregator problemAggregator) {
     if (repeat < 0) {
       throw new IllegalArgumentException("Repeat count must be non-negative.");
-    } else if (repeat == 1) {
-      return fromItems(name, List.of(item), null, problemAggregator);
     }
 
-    var builder = new InferredBuilder(repeat, problemAggregator);
     Object converted = item instanceof Value v ? Polyglot_Utils.convertPolyglotValue(v) : item;
```

Comment:
I think the `instanceof` check is now trivially true, not? Because `item` is `Value` by definition. So:
```suggestion
    Object converted = Polyglot_Utils.convertPolyglotValue(v);
```

Reply:
Done